Islandora/Alpaca
# GitHub actions

**GitHub Issue**: (https://github.com/Islandora/documentation/issues/1798)


# What does this Pull Request do?
Converts to github actions
updates the badges
removes travis

# What's new?
* new CI runner

# How should this be tested?
* make sure the tests pass (i don't think they will until this PR is merged - but you can see them running over on the branch itself)

Build completion proof here: https://github.com/asulibraries/Alpaca/runs/2479719378?check_suite_focus=true
Codecov proof here: https://codecov.io/gh/asulibraries/Alpaca/commit/35408a19672689883ecbe47a05827dc3583384db

# Notes
I am not sure what this section does: https://github.com/Islandora/Alpaca/blob/dev/.travis.yml#L24 so i don't know if i have to replicate this

# Interested parties
@Islandora/8-x-committers
